src/components/RoomPage/EditRoomTable.jsx

```diff
@@ -0,0 +1,180 @@
+                      <td className={`border-r px-4 py-2 text-center ${borderClass}`} style={{ verticalAlign: 'middle' }}>{row.description}</td>
+                      <td className={`border-r px-4 py-2 text-center ${borderClass}`} style={{ verticalAlign: 'middle' }}>{row.total_places}</td>
+                      <td className={`border-r px-4 py-2 text-center ${borderClass}`} style={{ verticalAlign: 'middle' }}>{row.type}</td>
+                      <td className={`border-r px-4 py-2 text-center ${borderClass}`} style={{ verticalAlign: 'middle' }}>{row.requirements}</td>
+                      <td className={`px-4 py-2 text-center ${borderClass}`} style={{ verticalAlign: 'middle' }}>
+                        <div className="flex items-center justify-center space-x-2">
+                          <button onClick={() => handleEdit(index)} className="flex items-center justify-center text-white rounded hover:bg-gray-200 w-[74px] h-[36px] p-0">
+                            <EditIcon className="w-full h-full" />
+                          </button>
+                          <button onClick={() => handleDelete(index)} className="flex items-center justify-center text-white rounded hover:bg-gray-200 w-[36px] h-[36px] p-0">
+                            <BinIcon className="w-full h-full" />
+                          </button>
+                        </div>
+                      </td>
+                    </tr>
+                );
+              })}
+              </tbody>
+            </table>
+          </div>
+        </div>
+        <div style={{ width: '1189px' }} className="flex pt-[43px] pb-[20px] pl-[25px] pr-[46px] bg-white rounded-[30px] mt-4 border border-gray-300">
+          <div className="flex-1">
+            <div className="mb-4 flex items-center">
+              <label className="block mr-2 w-32">Namn</label>
+              <input type="text" value={name} onChange={handleInputChange(setName)} className="w-[400px] h-[38px] px-3 py-2 border bg-[#D9D9D9BF]" />
+            </div>
+            <div className="mb-4 flex items-center">
+              <label className="block mr-2 w-32">Beskrivning</label>
+              <textarea value={description} onChange={handleInputChange(setDescription)} className="w-[400px] h-[38px] px-3 py-2 border bg-[#D9D9D9BF]" />
+            </div>
+            <div className="mb-4 flex items-center">
+              <label className="block mr-1 w-32">Antal sovplatser</label>
+              <input type="number" value={parseInt(beds)} onChange={handleInputChange(setBeds)} className="w-[400px] h-[38px] px-3 py-2 border bg-[#D9D9D9BF]" />
+            </div>
+            <div className="mb-4 flex items-center">
+              <label className="block mr-2 w-32">Typ</label>
+              <input type="text" value={type} onChange={handleInputChange(setType)} className="w-[400px] h-[38px] px-3 py-2 border bg-[#D9D9D9BF]" />
```

Comment:
Should be drop-down list. Room type can only be one of two values :  'room' or 'woman-only'. If free text users will make mistakes.